### DISCUSSION TITLE
Question on 'power-energy curves'

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#106] [Q&A: Analysis]
Hi ReEDS team,
I was wondering how to calculate the number in the Y-axis of the "power-energy curve" in Figure 32 from "Regional Energy Deployment System (ReEDS) Model Documentation: Version 2020". 

From the descriptive paragraph, it says: "We repeat this process for each season in each region over a wide range of storage capacity (from 0% to 90% of peak demand in 100 MW increments). The results of each dispatch are used to generate the "power-energy curve" in Figure 31, which allows us to calculate the marginal capacity credit for additional storage."

I think the x-axis is calculated by multiplying the number on the x-axis of the load (MW) in Figure 31 by the peak demand range of 0% to 90%. Is it correct? For any given point on the x-axis of the "power-energy curve", how is the y-axis determined? Thank you very much! 

![image](https://user-images.githubusercontent.com/78582696/157587161-490d04ed-b83d-407d-bbbc-208385f510d0.png)

![image](https://user-images.githubusercontent.com/78582696/157585501-f80688f7-8bac-4f46-aacd-babfb9b57ae9.png)

**Comment by wesleyjcole** [comment]
Figure 32 is created by reducing peak demand by 100 MW increments and determining how many MWh of storage are needed to get that reduction in peak demand.  For example, we determine how many MWh of storage are needed to get a 100 MW peak reduction, then how many MWh are needed for a 200 MW peak reduction, etc.  Figure 1 in the paper at https://www.sciencedirect.com/science/article/pii/S0306261920308977 helps to illustrate this concept.

**Reply by wanyingw193** [reply]
Thank you for the explanation. 

Does the curve in Figure 32 recalculate on each run and change depending on the different renewables' penetration percentage? Or is the supply curve included in the input file? Thanks for your guidance!

**Reply by wesleyjcole** [reply]
The curve in Figure 32 is recalculated after each solve (e.g., when running sequentially, it is calculated after each solve year).

**Reply by wanyingw193** [reply]
Thank you! Your answer is very helpful!

**Reply by wanyingw193** [reply]
Hi, I have a follow-up question about the power-energy curve in Figure 32. In that graph, it seems that short duration storage (2 hours) is always used to fill the peak load prior to longer duration storage (10 hours). 

What are the major reasons for this order of selection? Is this because short duration storage has a lower cost? Is there a situation where the model intends to select longer duration storage first and then short duration? Thanks!

**Reply by wesleyjcole** [reply]
We assume that short duration storage will always be used to meet the peak demand before longer duration storage.  We made that determination because it tends to be the most efficient way to meet peak demand.  Actual dispatch of storage will depend on the operational systems in place for a given system.  E.g., dispatch might be based on bids, heuristics, or some other rules.
